### PR TITLE
59 page adopt a jser

### PR DESCRIPTION
**Pull request by alinilea**

### CODE REVIEW

**Comment by alinilea** [comment]
contest-section: added section with commented code;
couch-surfing section: improved styling;
footer: improved styling and removed Become a Speaker link

File: app/components/common/rawHtml.js [hunk 1]
```diff
@@ -1,9 +1,21 @@
-const RawHtml = ({ content, className }) => (
-  <div>
+const RawHtml = ({ content, className, className1 }) => (
```
Comment (alexnm) [inline app/components/common/rawHtml.js]: maybe we can find a better naming for the classNames here

File: app/components/footer.js [hunk 2]
```diff
@@ -25,22 +25,24 @@ const Footer = () => (
             Buy Your Ticket
           </a>
         </div>
-        <div>
+        {/* <div>
           <a
```
Comment (alexnm) [inline app/components/footer.js]: I think it's pretty safe to remove the link, we won't need it anymore

File: app/components/updates/index.js [hunk 3]
```diff
@@ -8,20 +8,20 @@ const Updates = () => (
...
+
       <p>If you want to be &quot;adopted&quot; by one of our Cluj Javascripters during the JSHeroes conference, you can use our
-        <div>
-          <Link href="/adopt-a-jser">
-            <a title="Adopt a Javascripter couch-surfing program">
-              Adopt a Javascripter couch-surfing program
-            </a>
-          </Link>.
-        </div>
+        <br />
```
Comment (alexnm) [inline app/components/updates/index.js]: avoid using <br> to add spacing between paragraphs. In this scenario I think a wrapping div is easier and make the html more semantic

**Pull request merged**